deploy/examples/import-external-cluster.sh

```diff
@@ -152,6 +171,55 @@ function importRGWAdminOpsUser() {
   --from-literal=secretKey="$RGW_ADMIN_OPS_USER_SECRET_KEY"
 }
 
+function createRBDStorageClass() {
+cat << eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $RBD_STORAGE_CLASS_NAME
+provisioner: rook-ceph.rbd.csi.ceph.com
+parameters:
+  clusterID: $NAMESPACE
+  pool: $RBD_POOL_NAME
+  imageFormat: "2"
+  imageFeatures: layering
+  csi.storage.k8s.io/provisioner-secret-name: $CSI_CEPHFS_PROVISIONER_SECRET_NAME
+  csi.storage.k8s.io/provisioner-secret-namespace: $NAMESPACE # namespace:cluster
+  csi.storage.k8s.io/controller-expand-secret-name: rook-csi-rbd-provisioner
```

Comment:
WHat value should this contain if provisioner-secret-name is not always `rook-csi-rbd-provisioner`?
@Madhu-1 @leseb

Reply:
this also need to be same as the provisioner-secret-name